pom.xml

```diff
@@ -225,6 +225,11 @@
             <name>Mark Waite</name>
             <email>mark.earl.waite@gmail.com</email>
         </developer>
+        <developer>
```

Comment:
I believe the general pattern is that a "maintainer" is someone who actively reviews the submissions of others and has direct commit permission to the repository of this plugin.  Is that your intent?

This seems to be your only pull request to the git plugin repository, so I assume you don't intend to become an active maintainer of the plugin, rather preferring to contribute pull requests.